src/edge/edgeManager.ts

```diff
@@ -609,6 +609,22 @@ export class EdgeManager {
         return undefined;
     }
 
+    private async validateSolutionName(name: string, parentPath?: string): Promise<string | undefined> {
+        if (!name) {
+            return "The name could not be empty";
+        }
+        if (name.match(/[/\\:*?\"<>|]/)) {
+            return "Solution name cannot contain special characters";

```

Comment:
"Special characters" here is not clear enough. I prefer something like "Only *** are allowed".

Reply:
User can use Chinese or other utf8 characters for folder names, so I think "Only..." is not easy to describe the limitation

Reply:
How about change the message to Solution name can't contain characters: \/:*?<>|

Reply:
Thanks, fixed